torch/testing/_internal/schema_check_mode.py

```diff
@@ -72,68 +80,83 @@ def unwrap(e):
                     return e
             else:
                 return e
+
+        def parse_metadata(e):
+            if isinstance(e, torch.Tensor):
+                if not type(e) == torch.Tensor:
+                    try:
+                        current = e.elem
+                        return (deepcopy(current.stride()), current.storage()._cdata)
+                    except AttributeError as t:
+                        return e
+                else:
+                    return (deepcopy(e.stride()), e.storage()._cdata)
+            else:
+                return e
```

Comment:
nit: it looks like the value will never be used if e is not a tensor... if that's true could we return `None` instead?

Reply:
Yup, I had this from an old version where I was running into issues with having None instead, but it seems to work fine now